content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -193,6 +194,9 @@ the following types of volumes:
 * Azure Disk
 * Portworx
 * FlexVolumes
+* CSI
+
```

Comment:
Can this use a shortcode, ie `{{< feature-state for_k8s_version="v1.14" state="alpha" >}}`?

Reply:
(also, this might want to target _dev-1.14_ rather than _master_).

Reply:
changed the base to dev-1.14